pkg/e2e/infra/utils.go

```diff
@@ -265,8 +256,20 @@ func waitForClusterSizeToBeHealthy(client runtimeclient.Client, targetSize int)
 	}
```

Comment:
`waitForClusterSizeToBeHealthy` is not used anymore, not sure if we should keep it for future